SHARE-405 Added link to Huawei AppGallery

maxfazeli-dev opened the pull request:
Added Huawei AppGallery link to the welcome and help & tutorials page, as well as another entry to the footer.

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

HCrane commented on templates/Index/welcome-section.html.twig:
```diff
@@ -39,23 +39,31 @@
+          <a target="_blank" href="https://www.instagram.com/code_n_stitch/">
+            <div class="primary help-card d-flex align-items-center" style="flex-direction: column; justify-content: center;">
+              <div class="h1">
+                <img style="width: 60px; vertical-align:middle;" alt="Code&Stich Instagram" src="{{ asset('images/help/instagram.svg') }}">
```
The alt text should read `Code&Stitch`.

HCrane reviewed (approved):
LGTM :+1: IF you got the time please fix the Stitch typo :)

maxfazeli-dev committed:
SHARE-405 Added link to Huawei AppGallery

Added Huawei AppGallery link to the welcome and help & tutorials page, as well as another entry to the footer.
